packages/components/data-table/src/column-resizer.js

```diff
@@ -35,14 +34,13 @@ const DraggableArea = styled.div`
   }
 
   /* when dragging, we increase the area of the invisible draggable zone
-  to prevent accidentally hovering other elements which may cause flickers */
+  to prevent accidentally hovering other elements which causes flickering */
   &:active {
-    display: flex;
-    justify-content: center;
-    width: 50px;
-    right: -23px;
+    width: 20px;
```

Comment:
The area didn't have to become so large and with rounded values this is easier to understand.